front/src/modules/timesStops/utils.ts

```diff
@@ -49,3 +53,36 @@ export const formatSuggestedViasToRowVias = (
     };
   });
 };
+
+export function findNextScheduledPoint(
+  operationalPoints: OperationalPointWithTimeAndSpeed[],
+  tsPathPointsWithOpPointIndices: PathPointOpPointCorrespondance[],
```

Comment:
Why is there a ts prefix ? Also use pathStep instead of pathPoint. You can also rename the type PathPointOpPointCorrespondance
```suggestion
  pathStepsWithOpIndex: PathPointOpPointCorrespondance[],
```

Reply:
done